### PR TITLE
Fix go get newline

### PR DESCRIPTION
**Pull request by karthikraobr**
**What this PR does / why we need it**:
Fixes an issue with getting dependencies needed by the code generation step.
**Special notes for your reviewer**:

**If applicable**:
- [ ] All new Jobs, Commands, Executors, Parameters have descriptions
- [ ] If PR adds a new feature, Examples have been added
- [ ] README has been updated, if necessary

### CODE REVIEW

File: src/go/commands/make-gen.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ steps:
       name: '[go] install deps'
       command: |
         cd ../
-        go get -u github.com/vektra/mockery/cmd/mockery
-        golang.org/x/tools/cmd/goimports
+        go get -u github.com/vektra/mockery/cmd/mockery \
```
Comment (jeff-knurek) [inline src/go/commands/make-gen.yaml]: i think the slash should be `\`

Comment (karthikraobr) [reply src/go/commands/make-gen.yaml]: I'm sorry, I didn't get that. I guess I am using the same slash.

**Comment by TAvardit** [comment]
@jeff-knurek I am approving for now so we unblock `cmt-cmt` and `cmt-advertiserserv` and Kaarthik showed his changed worked here:
https://github.com/travelaudience/cmt-cmt/pull/595/files
If you still have comments we will fix it later.

File: src/go/commands/make-gen.yaml (hunk 1 above)
Comment (jeff-knurek) [reply src/go/commands/make-gen.yaml]: weird, I see that now, not sure what I saw the first time... 🤷‍♂

**Pull request merged**